.github/workflows/chart-release.yml

```diff
@@ -14,20 +15,17 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          submodules: recursive
 
-      - name: Configure Git
+      - uses: cachix/install-nix-action@v22
+      - name: Pre-populate nix-shell
         run: |
-          git config user.name "$GITHUB_ACTOR"
-          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+          nix-shell --pure --run "echo" ./shell.nix
```

Comment:
What is this here for? We're just using the action below

Reply:
Also here, are the chart metadata images list up to date already? I think so because this is just on push, not on tag?

Reply:
Oh yeah, I forgot to remove that. Have done that. 
yes.